src/scenes/endscreen.ts

```diff
@@ -30,17 +32,40 @@ export class EndScreen extends Phaser.Scene {
         this.scene.start("TitleScreen");
     }
 
-    private displayMessage(): void {
+    private displayByRole(): void {
+        const { showerOre, text } = this.displayData();
+        this.createShower(showerOre);
+        this.displayText(text);
+    }
+
+    private displayData() {
         let text: string = "You loose";
+        let showerOre: string = "coalNugget";
         const me: Nullable<Player> = PlayerManager.getMe();
         assert(me);
         if (
             (me.role === Role.Miner && SessionManager.getReachedGold()) ||
             (me.role === Role.Saboteur && !SessionManager.getReachedGold())
         ) {
+            showerOre = "goldNugget";
             text = "You win baby";
         }
+        return { showerOre, text };
+    }
+
+    private createShower(showerOre: string): void {
+        this.add.particles(0, 100, showerOre, {
+            x: { min: 0, max: ScreenWidth },
+            y: -400,
+            accelerationY: 500,
+            maxVelocityY: 700,
```

Comment:
Can you make it rain slower and stop after a certain time? I get nausy when i look at it for 3 seconds

Reply:
Let's talk about it on monday. Codecycle told me the opposite 😂

Reply:
sure lol

Reply:
i told that it should start earlier or not😅 but yes monday